pkg/rtc/transportmanager.go

```diff
@@ -241,7 +245,19 @@ func (t *TransportManager) RemoveSubscribedTrack(subTrack types.SubscribedTrack)
 
 func (t *TransportManager) SendDataPacket(kind livekit.DataPacket_Kind, encoded []byte) error {
 	// downstream data is sent via primary peer connection
-	return t.getTransport(true).SendDataPacket(kind, encoded)
+	err := t.getTransport(true).SendDataPacket(kind, encoded)
+	if err != nil {
+		if !errors.Is(err, io.ErrClosedPipe) && !errors.Is(err, sctp.ErrStreamClosed) && !errors.Is(err, ErrTransportFailure) && !errors.Is(err, ErrDataChannelBufferFull) {
```

Comment:
`!cloudutils.ErrorIsOneOf(err, etc...)`?

Reply:
Need to move that to OSS somewhere.

Reply:
...forgot where we were